NitroxClient/GameLogic/Spawning/Bases/BuildEntitySpawner.cs

```diff
@@ -1,6 +1,5 @@
 using System.Collections;
 using System.Collections.Generic;
-using System.Diagnostics;
```

Comment:
This is needed in debug. But I would let the line stay removed and change L27 to `System.Diagnostics.Stopwatch stopwach = ...`